openjdk8/src/test/resources/structure.yaml

```diff
@@ -0,0 +1,24 @@
+# Test spec for the Cloud Runtimes container structure testing framework.
+#
+# See https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/structure_tests/README.md for more detail.
+#
+schemaVersion: 1.0.0
+
+commandTests:
+- name: 'java version is 1.8'
+  command: ['java', '-version']
+  expectedError: ['openjdk version "1\.8.*"']
+
+fileExistenceTests:
```

Comment:
Can you also check the existence of `docker-entrypoint.bash` and `setup-env.bash`?